### PR TITLE
feat(jaeger): Bump to 2.50.1

### PR DESCRIPTION
**Pull request by gracedo**
**What problem does this PR solve?**:
Bump jaeger to latest 2.50.1

**Which issue(s) does this PR fix?**:

https://d2iq.atlassian.net/browse/D2IQ-99715

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: licenses.d2iq.yaml [hunk 1]
```diff
@@ -139,13 +139,13 @@ resources:
...
         url: https://github.com/istio/proxy
-  - container_image: docker.io/jaegertracing/all-in-one:1.46.0
+  - container_image: docker.io/jaegertracing/all-in-one:1.52.0
     sources:
       - license_path: LICENSE
         notice_path: NOTICE
         ref: v${image_tag}
         url: https://github.com/jaegertracing/jaeger
-  - container_image: docker.io/jaegertracing/jaeger-operator:1.46.0
+  - container_image: docker.io/jaegertracing/jaeger-operator:1.49.0
```
Comment (gracedo) [inline licenses.d2iq.yaml:148]: noticed the discrepancy and opened a fix upstream to use the expected version https://github.com/jaegertracing/helm-charts/pull/536

**Comment by gracedo** [comment]
https://github.com/mesosphere/kommander/pull/4209 is ✅

**Review by cwyl02 — APPROVED** [review]

**Comment by msdolbey** [comment]
@gracedo 

Don't know if this really matters - but mergebot didn't sign/verify a commit:
https://github.com/mesosphere/kommander-applications/pull/1863/commits/ebff8d973bdb69cdd923334d4e26744f99f0917e

**Review by msdolbey — APPROVED** [review]
👍 
Let's slam this

**Pull request merged**